poloTrader.py

```diff
@@ -30,18 +38,27 @@
 PASSWORD = secrets['emailSecret']
 
 TIMER = 600
+logging.info('TIMER set to ' + str(TIMER))
 # Boolean values
 PLOT_DATA = True
-REPEATE_SCRIPT = True
-SEND_SMS = True
-SEND_EMAIL = True
-VERBOSE = False
+REPEATE_SCRIPT = False
+SEND_SMS = False
+SEND_EMAIL = False
+VERBOSE = True
+DATA_PERIOD = 18000 #18000=5hrs, 86400=24hrs
+TIME_ZONE = 'Canada/Pacific'
```

Comment:
Maybe we can add this to the secrets file or something so we don't have to both keep changing the timezone